src/components/Workspace.jsx

```diff
@@ -210,6 +211,10 @@ class Workspace extends React.Component {
   }
 
   _renderOutput() {
+    if (includes(this.props.ui.minimizedComponents, 'preview')) {
```

Comment:
Since this is minimizing the output, rather than specifically the preview, I think we should be explicit about that and use the string `'output'` to designate it in the UI state.